### PR TITLE
Entrust deactivate resiliency

### PR DESCRIPTION
**Pull request by hosseinsh**
In Entrust test environment, one can issue test certificates. However, each certificate needs to be deactivate to free up the certificate issuance quota.
This improvement catches the corner-cases of a certificate already having been deactivated, or when the certificate cannot be found.

### CODE REVIEW

File: lemur/plugins/lemur_entrust/plugin.py [hunk 1]
```diff
@@ -293,12 +293,33 @@ def revoke_certificate(self, certificate, reason):
...
+            "external_id": f"{certificate.external_id}"
+        }
+
+        # Let's first check the status of the certificate
         base_url = current_app.config.get("ENTRUST_URL")
-        deactivate_url = f"{base_url}/certificates/{certificate.external_id}/deactivations"
-        response = self.session.post(deactivate_url)
-        metrics.send("entrust_deactivate_certificate", "counter", 1)
-        return handle_response(response)
+        deactivate_url = f"{base_url}/certificates/{certificate.external_id}"
```
Comment (jtschladen) [inline lemur/plugins/lemur_entrust/plugin.py:304]: Is this more like a `status_url`?

**Review by jtschladen — APPROVED** [review]

File: lemur/plugins/lemur_entrust/plugin.py (hunk 1 above)
Comment (hosseinsh) [reply lemur/plugins/lemur_entrust/plugin.py:304]: oh good catch, I think we can designate it as such. It fetches the entire certificate at this point, to avoid failures on deactivation.
